package.json

```diff
@@ -67,11 +67,11 @@
     "@babel/plugin-transform-runtime": "^7.13.7",
     "@babel/preset-react": "^7.16.7",
     "@folio/eslint-config-stripes": "^6.1.0",
-    "@folio/stripes": "^7.1.0",
+    "@folio/stripes": "^8.0.0",
```

Comment:
Hey @ryandberger , thanks for the PR! We already upgraded `stripes`, so the only thing left is `react-redux`